apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -923,7 +923,133 @@
+                        "max_share_target" : {
+                            "type" : "integer",
+                            "minimum" : 1,
+                            "maximum" : 115790322390251417039241401711187164934754157181743688420499462401711837020160
+                        },
+                        "desired_solve_time" : {
+                            "type" : "integer",
+                            "minimum" : 1,
+                            "maximum" : 180,
+                            "default" : 30
+                        },
+                        "max_solve_time" : {
+                            "type" : "integer",
+                            "minimum" : 1,
+                            "maximum" : 360,
+                            "default" : 60
+                        },
+                        "share_target_diff_threshold" : {
+                            "type" : "number",
+                            "default" : 5.0
+                        },
+                        "edge_bits" : {
+                            "type" : "number",
+                            "default" : 29
+                        },
+                        "max_jobs" : {
+                            "type" : "integer",
+                            "default" : 20
+                        },
+                        "max_workers" : {
+                            "type" : "integer",
+                            "default" : 20
+                        },
+                        "max_workers" : {
+                            "type" : "integer",
+                            "default" : 20
+                        },
+                        "msg_timeout" : {
+                            "type" : "integer",
+                            "default" : 15
+                        }
+                    }
+                },
+                "reward": {
+                    "type"    : "object",
+                    "additionalProperties": false,
+                    "properties" : {
+                        "reward_last_rounds" : {
+                            "type" : "integer",
+                            "default" : 2
+                        },
+                        "beneficiaries" : {
+                            "type" : "array",
+                            "items" : {
+                                "description" : "Public key receiving pool reward to percent share map.",
+                                "pattern": "^ak_[1-9A-HJ-NP-Za-km-z]*:[0-9]+(\.[0-9]*)?$"
+                            }
+                        },
+                        "keys" : {
+                            "type"    : "object",
+                            "additionalProperties": false,
+                            "properties" : {
+                                "dir": {
+                                    "description" : "Location (directory) of the public/private key pair used for signing of the payout contract call (relative to aestratum priv directory)",
```

Comment:
May benefit from a warning re safety of private key.

Reply:
I've added `The pool operator should ensure the node operates in trusted environment.` along with mention that the account's balance is not expected to be large, expect when there's software bug in distribution (which we are testing via CT).